app/Http/Controllers/Client/OrderController.php

```diff
@@ -23,21 +23,22 @@ public function __construct(OrderService $orderService)
      */
     public function index()
     {
-        $productData = $this->orderService->getProductSession();
+        $productSession = $this->orderService->getProductSession();
```

Comment:
thấy cái này hàm `getProductSession` này hơi dài dòng nhỉ
ở dưới e chỉ cần gọi `session('order_session')` là xong đó thôi

Reply:
để cái này để nó còn biết là session đó rỗng hay ko ạ, ko thì phần xử ở dưới của hàm index bị lỗi ạ

Reply:
```suggestion
        $productSession = session()->has('order_session') ? session('order_session') : [];
```
ví dụ như này chẳng hạn, nhiều cách xử lý mà